cpp/ps0002/p2Solution.cpp

```diff
@@ -3,17 +3,16 @@ using namespace std;
 
 
 int main() {
-	int building_residents = 50;
-	int drug_substitutions = 3;
-	int team_members = 8;
-	float baked_unit_price = 5.5;
-	char first_letter_of_name = 'A';
-	long int Chinese_birth_number = pow(10, 6);
-	double division_by_seven = 3.14286;
-	//cout << 22.0 / 7;
-	float factory_electricity_bill = 22000.500;
-	long long age_in_seconds = pow(50, 10);
-	//cout << age_in_seconds;
+	short building_residents = 50; // 0.5/0.75
+	short drug_substitutions = 3; // 0.75/0.75
+	short team_members = 8; // 0.75/0.75
+	float baked_unit_price = 5.5; // 0.75/0.75
+	char first_letter_of_name = 'A'; // 0.75/0.75
+	int Chinese_birth_number = 1000000; // 0.25/0.75
+	double division_by_seven = 3.14286; // 0.75/0.75
+	float factory_electricity_bill = 22000.50;  // 0.75/0.75
+	unsigned int age_in_seconds = 3000434323; // 0.25/0.75
+	// total 5.5/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables